coverage.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+echo "Executing coverage analysis script"
+# Programs below must be executed for gcov to generate .gcda files
+./src/tlog/tlog-rec-session -c whoami
+./src/tlog/tlog-rec
+./src/tlog/tlog-play
+lcov --compat-libtool --directory . --capture --output-file coverage.info
+lcov --remove coverage.info '*src/tltest*' '*lib/tltest*' '*include/tlog*' '*include/tltest*' '*usr/include/curl*' -o coverage.info
```

Comment:
Instead of collecting coverage data from all directories, and then removing data for some directories, could you instead collect it only from the directories we need? I.e. write 'lcov --compat-libtool --directory lib/tlog --directory src/tlog ...`?